packages/slap/slap.4.1.1/opam

```diff
@@ -0,0 +1,49 @@
+opam-version: "2.0"
+maintainer: "abe@sf.ecei.tohoku.ac.jp"
+authors: [ "Akinori ABE <abe@sf.ecei.tohoku.ac.jp>" ]
+license: "LGPL-2.1-only with OCaml-LGPL-linking-exception"
+homepage: "http://akabe.github.io/slap/"
+dev-repo: "git+https://github.com/akabe/slap.git"
+bug-reports: "https://github.com/akabe/slap/issues"
+build: [
+  [
+    "ocaml"
+    "setup.ml"
+    "-configure"
+    "--prefix"
+    prefix
+    "--disable-ppx" {ocaml:version < "4.02"}
+  ]
+  ["ocaml" "setup.ml" "-build"]
+  ["ocaml" "setup.ml" "-doc"] {with-doc}
+]
+install: ["ocaml" "setup.ml" "-install"]
+remove: [
```

Comment:
This is no longer needed with opam2, it keeps track of installed files on its own. I will remove it before merging, can you upstream the change?